src/app/loadout-builder/processWorker/process.ts

```diff
@@ -65,7 +65,9 @@ function insertIntoSetTracker(
             armorSetIndex < currentStatMix.armorSets.length;
             armorSetIndex++
           ) {
-            if (armorSet.maxPower > currentStatMix.armorSets[armorSetIndex].maxPower) {
+            if (
+              getPower(armorSet.armor) > getPower(currentStatMix.armorSets[armorSetIndex].armor)
```

Comment:
I pulled the maxPower attribute out of the loadouts so we have slightly less to calculate in the hot part of the process loop. This condition should rarely be triggered and we calculate it again when rendering.